cfme/configure/settings.py

```diff
@@ -184,273 +172,372 @@ def delete(self, cancel=False, *time_objs):
         view.flash.assert_no_error()
 
 
-@navigator.register(TimeProfileCollection, 'All')
-class TimeProfileCollectionAll(CFMENavigateStep):
-    VIEW = TimeProfileAllView
-    prerequisite = NavigateToAttribute('appliance.server', 'MySettings')
+class VisualForm(View):
 
-    def step(self):
-        self.view.mysetting.tabs.time_profile.select()
+    @View.nested
+    class grid_tile_icons(View):    # noqa
+        infra_provider_quad = BootstrapSwitch('quadicons_ems')
+        cloud_provider_quad = BootstrapSwitch('quadicons_ems_cloud')
+        host_quad = BootstrapSwitch('quadicons_host')
+        datastore_quad = BootstrapSwitch('quadicons_storage')
+        vm_quad = BootstrapSwitch('quadicons_vm')
+        template_quad = BootstrapSwitch('quadicons_miq_template')
+        long_text = BootstrapSelect('quad_truncate')
 
+    @View.nested
+    class start_page(View):    # noqa
+        show_at_login = BootstrapSelect('start_page')
 
-@navigator.register(TimeProfileCollection, 'Add')
-class TimeProfileAdd(CFMENavigateStep):
-    VIEW = TimeProfileAddView
-    prerequisite = NavigateToSibling('All')
+    @View.nested
+    class default_items_per_page(View):  # noqa
+        grid_view = BootstrapSelect('perpage_grid')
```

Comment:
**Optional**

Consider `grid_items`, `tile_items`, `list_items` since these aren't associated with 'view' changes like a view selector.

Reply:
That's the title of the control on the page, which is why I left those names.

![image](https://user-images.githubusercontent.com/28105554/37192361-4723019c-2322-11e8-8a0f-6faa379ae6cb.png)